doc/nrf/app_dev/device_guides/fem/21540ek_dev_guide.rst

```diff
@@ -46,13 +46,13 @@ Building for a multicore board
 
 When building for a board with an additional network core, like the nRF5340, add the ``-DSHIELD`` variable with the *childImageName_* parameter between ``-D`` and ``SHIELD`` to build for the network core.
 In this case, the application core is only responsible for forwarding to the network core the pins needed to control the nRF21540 EK.
-The application core can be set up to forward the needed pins using the ``-DSHIELD=nrf21540ek_fwd`` setting.
+The application core can be set up to forward the needed pins using the ``-DSHIELD=nrf21540ek`` setting.
 For example:
 
 .. parsed-literal::
    :class: highlight
 
-   west build -b nrf5340dk/nrf5340/cpuapp -- -DSHIELD=nrf21540ek_fwd -Dipc_radio_SHIELD=nrf21540ek
+   west build -b nrf5340dk/nrf5340/cpuapp -- -DSHIELD=nrf21540ek -Dipc_radio_SHIELD=nrf21540ek
 
 In this command, the *childImageName_* parameter has the ``ipc_radio_`` value as default and builds the application with support for the combination of 802.15.4 and Bluetooth.
 The ``ipc_radio`` has been used since the build system migration to sysbuild.
```

Comment:
I think this section needs a bigger rework.
1. It should explain that `-DSHIELD` is sufficient when there is only appcore and radiocore image without any other extra images, such as mcuboot. It should also explain when -DSHIELD can't be used and what to do instead.
2. The paragraph
    > In this case, the application core is only responsible for forwarding to the network core the pins needed to control the nRF21540 EK.

    is very nRF53 specific. Newer SoCs won't have this problem. Perhaps it could be changed to
    > In the case of nRF5340, the application core is responsible for forwarding the pins needed to control the nRF21540 EK to the network core.
    
    This information is needed only when the user can't use `-DSHIELD`, so it should be put into the context somehow.
 3. `-Dipc_radio_SHIELD=nrf21540ek` could be removed. A different example build command could be provided for the case, where `-DSHIELD` can't be used.